docs/appendices/release-notes/unreleased.rst

```diff
@@ -42,6 +42,9 @@ Unreleased Changes
 Breaking Changes
 ================
 
+- Changed the error code for the psql protocol when a document exists
```

Comment:
Not sure if we need to list this as breaking change - I think moving it to changes would be good enough. In a way it could even be considered a bug that we didn't return the right error code, so moving it to change may be a good compromise